sweetest/src/main/java/com/mysugr/sweetest/framework/build/BaseBuilder.kt

```diff
@@ -83,21 +144,39 @@ abstract class BaseBuilder<TSelf, TResult : BaseAccessor>(
         testContext.factories.configure(FactoryRunner0(R::class.java, createObject))
     }
 
-    inline fun <reified T : Steps, reified R : Any> offerFactory(noinline createObject: (T) -> R) = apply {
-        testContext.factories.configure(FactoryRunner1(R::class.java, T::class.java, createObject))
-    }
+    inline fun <reified T : Steps, reified R : Any> offerFactory(noinline createObject: (T) -> R) =
```

Comment:
i think auto format changed the line breaks. can you change the code style settings in your IDE to 120 characters and revert these changes?